.github/workflows/npm-checks.yml

```diff
@@ -1,24 +0,0 @@
-name: NPM Checks
-
-on:
-  pull_request:
-    branches: [master, shutdown-master]
```

Comment:
I don't think we have a `master` branch here, just `main`, but I am not sure about this `shutdown-master` value - I omitted it in the pr-checks.yml file for now, but I can add it back if needed.